packages/rocketchat-lib/server/models/_BaseCache.js

```diff
@@ -552,6 +552,23 @@ class ModelsBaseCache extends EventEmitter {
 			fieldsToGet.push('_id');
 		}
 
+		let pickFields = (obj, fields) => {
+			let picked = {};
+			fields.forEach((field) => {
+				if (field.indexOf('.') !== -1) {
+					let nestedFields = field.split('.');
```

Comment:
I looked for something to that but didn't found. I'll look for this `objectPath` :wink: